scanpipe/pipes/docker.py

```diff
@@ -123,6 +123,7 @@ def scan_image_for_system_packages(project, image, detect_licenses=True):
     for i, (purl, package, layer) in enumerate(installed_packages):
         logger.info(f"Creating package #{i}: {purl}")
         created_package = pipes.update_or_create_package(project, package.to_dict())
+        created_package.update_extra_data({"image_id": image.image_id})
```

Comment:
Can you elaborate on why you need to keep this?

Reply:
Also, in that shape, this will trigger 1 extra SQL query per packages.

Reply:
The pipeline needs to know package's alpine version (It is stored in `project.extra_data["images"][...]["distro"]["version_id"]`).